src/cmd/cmd_sym.ml

```diff
@@ -41,27 +41,40 @@ let run_file ~entry_point ~unsafe ~rac ~srac ~optimize ~invoke_with_symbols pc
   let c = Interpret.Symbolic.modul link_state.envs m in
   Choice.bind pc (function Error _ as r -> Choice.return r | Ok () -> c)
 
-let print_bug model_output_format no_value no_assert_failure_expression_printing
-    =
+let print_bug model_format model_out_file id no_value no_stop_at_failure
+  no_assert_failure_expression_printing =
   let to_string =
-    match model_output_format with
+    match model_format with
     | Cmd_utils.Json -> Smtml.Model.to_json_string
     | Scfg -> Smtml.Model.to_scfg_string ~no_value
   in
+  let to_file path model =
+    let path =
+      if no_stop_at_failure then Fpath.(path + "_" + string_of_int id) else path
```

Comment:
It should be mentioned in the manpage that the path given on CLI will be used as a prefix when there are many test cases